### ISSUE TITLE
cross-entropy loss for multiclass classification

### ISSUE THREAD

**Issue by Blubberli** [Blubberli/ambiguous-alpaca#6]
- implement the cross-entropy loss for multiclass classification
- use torch.nn.functional.cross_entropy, this does softmax and loss computation in one go, expects unnormalized class scores
